src/pages/signup-affiliates/components/_account-terms.tsx

```diff
@@ -138,7 +141,15 @@ const AccountTerms = ({ affiliate_terms_data, updatedData, onValidate }: Account
                             handleChangeCheckbox={(value) => handleChange(value, name)}
                             link_text={link_text}
                             color="var(--color-black-3)"
-                        />
+                        >
+                            <StyledLink
+                                external="true"
```

Comment:
Can we pass `boolean` value to `external` property instead of `string`?

Reply:
By default its true..so i removed it.thanks